packages/sources/coinmetrics/src/adapter.ts

```diff
@@ -78,6 +78,10 @@ export const makeWSHandler = (config?: Config): MakeWSHandler => {
           url,
         }
       },
+      shouldRetryConnection: (closeContext) => {
```

Comment:
Do we know the shape of the error message that we can document here as an interface?